k8s/sandbox/common/neuvector/nv-storage-secret.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  creationTimestamp: null
+  name: nv-storage-secret
+  namespace: neuvector
+spec:
+  encryptedData:
+    azurestorageaccountkey: AgA5woZz57GZpTQDJAvbTpr+66gW5hA09qLHk2fZRZwK+D3VlBoGGRxiyM0aS5Ei0A0Q9JxT+gpdRj5tfN4oJKJH2cJDhrvRiG4ct7e8ZGD+/PX9rFjYH9Em4ZMrLVqmfVtNmFAGopVomxlhTcB3q61w3ny3qJBn/ZnBh+nY1xISDaHwZEnKk9cBe96BKaVXiZ2U9hD5c/AFFyDVSeP1h+FMdFUyPyR7B5BK7nfRgRm2oNivPboOCaRMykvFViq3NucrhTtpnsEDM7o5XvF8fyljP1JODqzW3C/V3eFEQb2/kEUfqTUloQ1tv09jY9mLIO+segVK8S/8i5HB7JPQVsBlbEdQsUi6sjZd9wWlgSXUnA8F7BBCOZI/altTvN5dybumu/Kot0t4A8AtBymQKm3GhWp6bJLJFlzvbm2YowV4zG/hYmyENCkRrf3sTfsG2tKcPI6POb0IWd2JiaRS3vOUyrAoCx9AXAm3dSkYYAGfFKkE4Ak4WB3cuHhku4pBqa7fSIifl6DS1kSsTxuXWFHrjdvjyLDhvve6zFlcTwPo0AeBXVd+V+4M+xIUy5Wgwd4dqkJJnIyN1exShyhONHrdg8F++FUY9QUO05xaLG8fu36KYMIbV7LjO8qdiDjx86c8V0KP/JEL5WedUzvZCQjDwxQeoqbosxJ/V8VKTWHTzB+jhhRTo8WKKUR6IT9kr7YRM2Q4SwKC8kxzgq10YpZr7E1zj9WjOA9qExMLz4i3S6Hs4sUVn2tKHWkXJXijc/Fxz6Q6kwbLDTH1fjYcXJRw9HpULQ/Qi5DGDBfjSCcBK6pG2XUv5PzD
```

Comment:
is there a storage account in sbox ? hope u are not using the one from aat.

Reply:
Hey, yeah - I ran through this https://github.com/hmcts/cnp-flux-config/blob/10a9625bb9166e4217db22c379c1089ac321d594/docs/new-cluster.md#neuvector for cftappssbox storage account, - it's not copy pasted from somewhere else.